standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1963,6 +1964,18 @@ public enum ConfVars {
       this.description = description;
     }
 
+    ConfVars(String varname, String hiveName, String defaultVal, boolean caseSensitive,
```

Comment:
can we reuse code in the overloaded constructors instead of copy&paste?

Reply:
that's not a pattern, but a bad coding style

Reply:
agree, but that needs to be refactored in a separate PR that will take care of all ctors.

Reply:
ok, agree